### PR TITLE
Create nejm-catalyst.csl

### PR DESCRIPTION
**Pull request by badgettrg**
I left authors of NEJM style this was based on as contributors. I can't make the 'access' macro display for journal articles.

### CODE REVIEW

File: nejm-catalyst.csl [hunk 1]
```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-US">
+  <info>
+    <title>NEJM Catalyst</title>
+    <title-short>NEJM Catalyst</title-short>
+    <id>http://www.zotero.org/styles/nejm-catalyst</id>
+    <link href="http://www.zotero.org/styles/nejm-catalyst" rel="self"/>
+    <link href="https://catalyst.nejm.org/pb-assets/images/NEJM%20Catalyst%20Reference%20style-1589400257323.pdf" rel="documentation"/>
```
Comment (rmzelle) [inline nejm-catalyst.csl:8]: @badgettrg, thanks. Where did you find this PDF link? I only found links to https://catalyst.nejm.org/pb-assets/author-center/ReferenceswLinks_Jul2020-1595613582897.pdf at https://catalyst.nejm.org/author-center/article-types, although that seems to describe the same style.

**Comment by badgettrg** [comment]
The pdf I used, https://catalyst.nejm.org/pb-assets/images/NEJM%20Catalyst%20Reference%20style-1589400257323.pdf , is linked from the page https://mc05.manuscriptcentral.com/catalyst (screen capture attached) in manuscript central after you authenticate there.

Any tips on how to make the XML display both the publisher's URL and the URL via doi from type=article-journal as indicated in the Catalyst style?

thanks
________________________________

**Comment by rmzelle** [comment]
> Any tips on how to make the XML display both the publisher's URL and the URL via doi from type=article-journal as indicated in the Catalyst style?

What are you testing the style with? If you're using Zotero, you might have to check "Include URLs of paper articles in references" in the preferences (https://www.zotero.org/support/preferences/cite).

**Comment by badgettrg** [comment]
Yes, Zotero. Thanks that helps much. I thought a setting must be lurking somewhere that oversides the XML logic.
________________________________

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**